packages/wp-source/index.ts

```diff
@@ -8,6 +8,7 @@ export type Pattern<F extends Function = (params: any) => any> = {
   priority: number;
   pattern: string;
   func: F;
+  isRequired?: boolean;
```

Comment:
Why do you need `isRequired`? When is it used? (I don't see it in the rest of the PR)

Reply:
Oops, I forgot to delete that line.